work-process/scrum.md

```diff
@@ -79,9 +79,22 @@ know we are paddling hard, but are we all paddling in the right direction?
 
 A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story.
 
-One full time member on sprint should be able to do 10 Story Points. Estimates are based on [ZenHub estimates](https://www.zenhub.com/blog/software-estimates/) and the `Estimate` label in a User Story also shows available values.
+One full time member on sprint should be able to do 10 Story Points. Estimates are based on [ZenHub estimates](https://www.zenhub.com/blog/software-estimates/).
 
-In the *Vote* step, the Scrum Master updates the User Story with a `✋ [vote]` prefix to the title and assigns the team members from whom they want to receive Story Point estimations. Once a team member recieves a notification and gives their Story Point estimation, they should unassign themselves from the User Story.
+In the *Vote* step, the Scrum Master updates the User Story with a `✋ [vote]` prefix to the title and assigns the team members from whom they want to receive Story Point estimations.
+
+Team members vote on a story by adding a comment with their story point estimation. To keep the estimate hidden so as to not influence anyone else's voting, we use the following code in a GitHub comment. Add this snippet to GitHub [saved replies](https://github.com/settings/replies) to reuse it.
+
+```
+<details>
+  <summary>Story Point estimate</summary>
+  SP:
+</details>
+```
+
+The possible story points values can be seen in the `Estimate` label drop-down on the right-hand side of a GitHub issue.
```

Comment:
why not add all the possibilities in the code above?

Reply:
>Add this snippet to GitHub [saved replies](https://github.com/settings/replies) to reuse it.

I would move this sentence after the code since the former sentence indicates that the code comes after it.